download_posts.py

```diff
@@ -62,18 +65,30 @@ def f(field):
         'current_mood': f('current_mood')
     }
 
+def increment_month(date):
+    return date + relativedelta(months=1, day=1)
+
 def download_posts(cookies, headers):
     os.makedirs('posts-xml', exist_ok=True)
     os.makedirs('posts-json', exist_ok=True)
 
+    # Get start and end dates with validation
+    start_date, end_date =  get_date_input("Enter start date in YYYYMMDD format: ")
```

Comment:
I don’t see how this text "Enter start date in YYYYMMDD format: " is used anywhere